de_web_plugin.cpp

```diff
@@ -93,6 +93,7 @@ const quint64 bjeMacPrefix        = 0xd85def0000000000ULL;
 const quint64 xalMacPrefix        = 0xf8f0050000000000ULL;
 const quint64 lutronMacPrefix     = 0xffff000000000000ULL;
 const quint64 legrandMacPrefix    = 0x0004740000000000ULL;
+const quint64 aqaraOppleMacPrefix = 0x04cf8c0000000000ULL;
```

Comment:
According to https://wintelguy.com/index.pl the MAC address prefix `04cf8c` belongs to XIAOMI Electronics,CO.,LTD.
I think Opple is just product line (?), perhaps a name like `xiaomiMacPrefix` or `aqaraMacPrefix` is more general and future safe.

Reply:
Well, there are already Aqara devices which use a different MAC prefix, so i guess Xiaomi uses this new MAC prefix for the Aqara Opple line?

Reply:
Seems so. I'd prefer xiaomiMacPrefix

Reply:
Xiaomi devices (including some Aqara ones) are already under the jennicMacPrefix. I have no idea why this is called like that. 
The Aqara Opple devices have this new mac prefix. 
Calling it just `xiaomiMacPrefix` or `aqaraMacPrefix ` will make it even more confusing then it already is...

Reply:
Im fact, it's not really difficult at all.

We're here talking about the manufacturer of the transmission unit or maybe even SoC. Not too many vendors manufacture them but regardless, every one of them gets a unique MAC prefix for identification. Now, those transmission units or SoCs are used to build devices. Maybe the MAC vendor builds them or a third party vendor. Based on those possibilities, you cannot stipulate only Xiaomi uses that MAC for their devices. Maybe they sell them... To make things worse, there are also OEMs, pretty much selling the hardware but the buying company labels them as their own...

Bottom line is that you cannot do anything about the confusion. That's the reason the whilelist combinations are in the code, as of now. From my point of view, there is only one reasonable decision on how to name it.

Reply:
That is true but it is not related to this. I am not trying to fix the confusion here, I am trying to prevent further confusion. The Aqara Opple devices have clearly an unique mac and should be grouped smorodiny to that.

Reply:
I'm afraid your answer tells that you still did not get the idea. Maybe it will me more obvious if in a couple of months, e.g. IKEA devices come up with a Xiaomi MAC...

Anyhow, I stop arguing here and leave the decision up to @manup

Reply:
No worries :) there is kind of a system behind all this. The Jennic prefix refers to the underlying radio manufacturer MAC address prefix. Companies often start using  such MAC address prefixes before getting their own prefix (sometimes multiple).

The Jennic prefix and radios are not just used by Xiaomi, same goes for emberMacPrefix, tiMacPrefix, stMacPrefix and silabsMacPrefix. These are used by multiple companies because they are using their radios of the respective manufacturers.

Another example is the philipsMacPrefix, which refers to Philips as company (not just the Hue line) `0x001788`. I think xiaomiMacPrefix is the most adequate name and I guess there will be new Zigbee products from Xiaomi / Aqara with this prefix not necessarily belonging to the OPPLE range.

Reply:
Thanks! That makes sense now. I was kind of confused by the jennic mac prefix since all other xiaomi/aqara devices currently  are using that, so i was not sure how this new switches fit in that. I will rename the Opple mac prefix to `xiaomiMacPrefix `.